### ISSUE TITLE
About：  Configuration file for elegoo-Neptune machine

### ISSUE THREAD

**Issue by NARUTOfzr** [Toylerrr/ELEGOO_Neptune2_Cura#5]
Hello friend, I'm ZIRONG-FENG(冯子荣) from ELEGOO tech support team. Sorry if I have bothered you. 
Our team have noticed that you have upload ELEGOO printer files to Cura and we appreciate it very much.
I'd like to ask if there is a possibility to help us add the ELEGOO printer model profiles that we have corrected to Ultimaker Cura. 

Looking forward to your reply.

Sincerely,
ZIRONG-FENG

**Comment by Toylerrr** [comment]
Yes I would love to help! Let me know how I can help. Currently the project pushed to the Cura master is very barebones and left a lot of the updates out for simplicities sake. My next move is to merge this project into the Cura master project.

**Comment by NARUTOfzr** [comment]
I'm glad you're willing to help. Thank you very much! So far I have created a pull request to [Cura: main] from ELEGOO's machine configuration file, and I have modified and replaced the configuration file you created. I hope you don't mind. Because I was the first time request, and I don't know whether can succeed, this is my request link: [https://github.com/Ultimaker/Cura/pull/12656](https://github.com/Ultimaker/Cura/pull/12656) if you find any wrong place, you are welcome to point out and help to correct, sincerely thank you!!!!!
